src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Figures {

```

Comment:
```suggestion
public abstract class Figure {
```

класс Figure должен быть абстрактным, так как мы не можем создать обхект класса фигура. В этом задании нам нужно создать 2 интерфейса, которые будут задавать поведение наших фигур - Drawable и AreaCalculator например(с методами draw() и getArea() соответственно). Эти интерфейсы мы должны импелементировать нашим абстрактным классом Figure, так как абсолютно все фигуры должны иметь это поведение.